src/modules/75-cf/components/SectionToggle/SectionToggle.tsx

```diff
@@ -21,7 +21,11 @@ const SectionToggle = (): ReactElement => {
   const Item = ({ link, text, tooltipId }: { link: string; text: StringKeys; tooltipId: string }): ReactElement => (
     <li className={css.item}>
       <NavLink to={`${link}?activeEnvironment=${activeEnvironment}`} className={css.link} activeClassName={css.active}>
-        <StringWithTooltip stringId={text} tooltipId={tooltipId} />
+        <StringWithTooltip
+          stringId={text}
+          tooltipId={tooltipId}
+          onClick={(e: React.MouseEvent<HTMLElement, MouseEvent>): void => e.stopPropagation()}
```

Comment:
I think if you make the change in the interface you can simplify this...
```suggestion
          onClick={e => e.stopPropagation()}
```